src/emails/emailConfirmation.tsx

```diff
@@ -34,6 +34,7 @@ export const confirmEmail = (props: EmailTemplateProps) => (
             />
           </Section>
           <Heading style={h1}>Confirm your email address</Heading>
+          <button> Confirm Email</button>
```

Comment:
We should have written more clear instructions. Sorry about that! 

This will actually need to be a link instead of a button, although it will be styled like a button. You can import the `Link` component from `'@react-email/components'. Then you'll need to pass the link from the props as the `href`. 

Here's an example of this link button that we can use.

```html
    <Link
      	  href={props.link}
          className="rounded-lg border bg-teal-500 px-5 py-3 text-xl text-white no-underline"
        >
          Confirm Email
    </Link>
```

Lastly, we should add some vertical padding surrounding the button by putting the button inside of a <Section> component, also imported from '@react-email/components'.

Full code snippet:

```html
<Section className=" py-6">
  <Link
    href={props.link}
    className="rounded-lg border bg-teal-500 px-5 py-3 text-xl text-white no-underline"
  >
    Confirm Email
  </Link>
</Section>
```

Reply:
Since we are adding this button. We should also update the text below it (above the link) to the following: 

` If you are having trouble clicking the button, copy and paste the URL below 👇`